hokusai/production.yml

```diff
@@ -74,7 +74,11 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-ssl-ports: '443'
     service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
     service.beta.kubernetes.io/aws-load-balancer-connection-draining-enabled: 'true'
-    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '300'
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '5'
```

Comment:
Also, it seems like the node [default KeepAliveTimeout is 5 seconds](https://nodejs.org/docs/latest-v10.x/api/http.html#http_server_keepalivetimeout). Would we want to ensure that this timeout is something _less than_ 5, like 4? I think then we would systematically avoid these 504s, right?

Reply:
Yes I just worry about setting this too low then killing in-flight requests.  Perhaps it's better to raise Node's timeout to 30 and set the ELB's at 29?

Reply:
Hmm - I agree in spirit, but just worry about the possible ramifications of tuning node directly like this. We have a solution that we think works (although maybe doesn't prohibit all 504s), so I'd actually roll with this. Effectively, I retract my first comment :)